index.js

```diff
@@ -1,73 +1,111 @@
-    const removeBtn = document.createElement('button');
-    removeBtn.classList.add('remove-btn');
-    removeBtn.textContent = 'Remove';
-    removeBtn.addEventListener('click', () => removeBook(index));
+    const title = titleInput.value;
+    const author = authorInput.value;
 
-    bookDiv.appendChild(removeBtn);
-    bookListDiv.appendChild(bookDiv);
-  });
-}
+    if (title && author) {
+      const newBook = {
+        title,
+        author,
+      };
 
-// Function to add a new book to the collection
-function addBook(event) {
-  event.preventDefault();
+      this.books.push(newBook);
 
-  const titleInput = document.getElementById('title');
-  const authorInput = document.getElementById('author');
+      localStorage.setItem('bookCollection', JSON.stringify(this.books));
 
-  const title = titleInput.value;
-  const author = authorInput.value;
+      titleInput.value = '';
+      authorInput.value = '';
 
-  if (title && author) {
-    const newBook = {
-      title,
-      author,
-    };
-
-    bookCollection.push(newBook);
+      this.displayBooks();
+    }
+  }
 
-    // Save the updated book collection to localStorage
-    localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+  removeBook(index) {
+    this.books.splice(index, 1);
 
-    // Reset input fields
-    titleInput.value = '';
-    authorInput.value = '';
+    localStorage.setItem('bookCollection', JSON.stringify(this.books));
 
-    displayBooks();
+    this.displayBooks();
   }
 }
 
-// Event listener for form submission
-const addBookForm = document.getElementById('addBookForm');
-addBookForm.addEventListener('submit', addBook);
+const bookCollection = new BookCollection();
 
-// Display the books initially
-displayBooks();
+const addBookForm = document.getElementById('addBookForm');
+addBookForm.addEventListener('submit', (event) => bookCollection.addBook(event));
+
+bookCollection.displayBooks();
+
+// Displaying the navitems individually
+
+const mainLink = document.getElementById('Main-Link');
+const addLink = document.getElementById('Add-Page-Link');
+const contactLink = document.getElementById('Contact_Link');
+
+const awesomeBooks = document.querySelector('.books-content-top');
+const contactPage = document.querySelector('.contact-container');
+const formPage = document.querySelector('.form-container');
+
+mainLink.addEventListener('click', () => {
+  mainLink.classList.toggle('clicked');
+  contactLink.classList.remove('clicked3');
+  addLink.classList.remove('clicked2');
+  formPage.classList.add('less');
+  formPage.classList.remove('remove');
+
+  contactPage.classList.remove('add');
+  awesomeBooks.classList.add('more');
+});
+
+addLink.addEventListener('click', () => {
+  addLink.classList.toggle('clicked2');
+  mainLink.classList.remove('clicked');
+  contactLink.classList.remove('clicked3');
+  awesomeBooks.classList.remove('more');
+  contactPage.classList.remove('add');
+  formPage.classList.remove('less');
+});
```

Comment:
Good use of `add` and `remove` `classList` methods of hiding and displaying your page items. 🚀 However there is one tiny change I would suggest you make to complete your application. ♻️ 

- [ ] Kindly make sure that all page elements appear after their corresponding navigation link is clicked. 🖱️ Currently, your `Add page` link does not display the `Add Book` form after the `Contact` link has been clicked. 

### Please check as shown in the GIF below ⬇️ 
![missing book form](https://github.com/Mike47ip/Awesome_Book/assets/117144028/f3eb5782-590a-421f-b3c2-23c7ed335039)

I suggest that you remove that `class` that hides the form when the `Contact` link is clicked. Consider the code below ⬇️ 
  `formPage.classList.remove('less', 'remove');`